BookingQueueSubscriber/BookingQueueSubscriber.UnitTests/BookingQueueSubscriberFunctionTests/HearingIsReadyForVideoIntegrationMessageTests.cs

```diff
@@ -215,6 +216,78 @@ public async Task should_process_a_single_day_hearing_ready_event_with_a_judge_a
+    [Test]
+    public async Task should_process_a_single_day_hearing_ready_event_with_a_judge_and_participants_feauture_toogle_new_template_on()
+    {
+      _featureToggles.PostMayTemplateToggle = true;
+        const string message = @"{
+  '$type': 'BookingsApi.Infrastructure.Services.IntegrationEvents.EventMessage, BookingsApi.Infrastructure.Services',
+  'id': '1f690c34-353b-4d8b-99a3-e9240088faff',
+  'timestamp': '2023-10-25T13:13:10.8432746Z',
+  'integration_event': {
+    '$type': 'BookingsApi.Infrastructure.Services.IntegrationEvents.Events.HearingIsReadyForVideoIntegrationEvent, BookingsApi.Infrastructure.Services',
+    'hearing': {
+      '$type': 'BookingsApi.Infrastructure.Services.Dtos.HearingDto, BookingsApi.Infrastructure.Services',
+      'hearing_id': 'a562fc34-6b0a-4ca0-a963-46678ac4573d',
+      'group_id': null,
+      'scheduled_date_time': '2023-10-30T12:00:00Z',
+      'scheduled_duration': 480,
+      'case_type': 'Asylum Support',
+      'case_number': 'test Marco 6 10204',
+      'case_name': 'Pippo6 10204',
+      'hearing_venue_name': 'Bedford County Court and Family Court',
+      'record_audio': true,
+      'hearing_type': 'Adjourned/Resumed Hearing'
+    },
+    'participants': [
+      {
+        '$type': 'BookingsApi.Infrastructure.Services.Dtos.ParticipantDto, BookingsApi.Infrastructure.Services',
+        'participant_id': '068f8ff6-0800-4cfa-9640-dd5eee0ac744',
+        'fullname': ' Manual Judge_06',
+        'username': 'manual.judge_06@hearings.reform.hmcts.net',
+        'first_name': 'Manual',
+        'last_name': 'Judge_06',
+        'contact_email': 'manual.judge_06@hearings.reform.hmcts.net',
+        'contact_telephone': '011234556789',
+        'display_name': 'Manual Judge_06',
+        'hearing_role': 'Judge',
+        'user_role': 'Judge',
+        'case_group_type': 'judge',
+        'representee': '',
+        'linked_participants': [],
+        'contact_email_for_non_e_jud_judge_user': null,
+        'contact_phone_for_non_e_jud_judge_user': null,
+        'send_hearing_notification_if_new': true
+      },
+      {
+        '$type': 'BookingsApi.Infrastructure.Services.Dtos.ParticipantDto, BookingsApi.Infrastructure.Services',
+        'participant_id': '5eb27388-7e02-4ab7-8dca-e70a0e606746',
+        'fullname': 'Mr Ma Gagliardi',
+        'username': 'ma.gagliardi@hearings.reform.hmcts.net',
+        'first_name': 'Ma',
+        'last_name': 'Gagliardi',
+        'contact_email': 'marco6@email.com',
+        'contact_telephone': '+4412346786',
+        'display_name': 'marco6 rep',
+        'hearing_role': 'Representative',
+        'user_role': 'Representative',
+        'case_group_type': 'appellant',
+        'representee': 'marco',
+        'linked_participants': [],
+        'contact_email_for_non_e_jud_judge_user': null,
+        'contact_phone_for_non_e_jud_judge_user': null,
+        'send_hearing_notification_if_new': true
+      }
+    ],
+    'endpoints': []
+  }
+}";
```

Comment:
it might be worth asserting which emails are expected to be sent with this test too

Reply:
done

Reply:
add a LIP participant to confirm the correct email is being sent too

Reply:
test added